iosApp/iosApp.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
@martyu Every time I open AppCode to ensure I didn't break anything on the iOS app I get this file updated, so I added it for now? If it causes any issues on your side I can remove it through .gitignore

Reply:
Not sure why I don't see it, but yeah should be fine